### DISCUSSION TITLE
Make Collection Actuator Also Disables/Enables Child Collections

### DISCUSSION THREAD

**Discussion by MakataUrugami** [UPBGE/upbge#1660] [Ideas/Features for UPBGE 0.3+]
I'm making a game with the map splitted in collections, I have created a collection for each part of the map, but also a sub collections like "Furniture, Decals, Building" ones. But it seems that it doesn't disable those child collections, resulting on me creating empties with message listeners to disable a whole group of collections.

Any chances of it to be done?

**Comment by rpaladin** [comment]
Are you perhaps looking for the `Set Collection Visibility` node?
Don't forget to check the `Include Children` option if you want all of a collection's sub-children to be included in the visibility toggle process.

![image](https://user-images.githubusercontent.com/69180012/155584789-89b83468-a628-462a-ba3b-c4ae24b16080.png)

**Reply by MakataUrugami** [reply]
Hmm... I actually haven't thinking about using logic nodes, but it indeed could solve my problem. Thanks.
